code/modules/clothing/suits/jobs.dm

```diff
@@ -76,7 +76,7 @@
 //Merchant
 /obj/item/clothing/suit/storage/qm_coat
 	name = "guild merchant coat"
-	desc = "An ideal choice for a smuggler. This coat seems have good impact resistance, and is made from resistant and expensive materials."
+	desc = "A fabulous black coat, perfect for a steeled capitalist."
```

Comment:
```suggestion
	desc = "Fabulous black coat, perfect for a steeled capitalist."
```
Leaving out "A" and "An" at the start of item descriptions seems to be a pattern, let's keep things consistent here.
Can't 100% tell how grammatically correct it is though.

Reply:
I thought that was the correct consistency, from looking at other items